src/lib/watcher/runner.ts

```diff
@@ -126,25 +119,51 @@ register({
     IPC.client.senders.moduleImported({ filePath })
   })
 
-  if (!process.env.NEXUS_EVAL) {
-    throw new Error('process.env.NEXUS_EVAL is required')
-  }
+  const startModuleFileName = layout.sourceRoot + '/index.ts'
```

Comment:
Might be good to use `Path.join` and not use `/`? I know there are other places where we also use `/`, but unless I'm wrong, all of these paths won't work on Windows, will they?